CMakeLists.txt

```diff
@@ -34,6 +37,8 @@ target_include_directories(named_tuple INTERFACE
   $<INSTALL_INTERFACE:include>
   )
 
+target_sources(named_tuple INTERFACE ${named_tuple_SOURCE_DIR}/include/named_tuple.h)
```

Comment:
This makes it more IDE-friendly by showing the header file as part of the project in IDEs.